### PR TITLE
removing travis references

### PR DESCRIPTION
**Pull request by mikebz**
Travis doesn't seem to be used by kustomize any more, yet we have a directory named travis as well as a script that allows an early travis exit.

This PR is cleaning up the naming and removing scripts that I believe are not used any more.

## Testing ##
The CI will test the changes once the PR is submitted
Did a grep on the source for `travis` - nothing found any more.
Checked a recent existing PR: https://github.com/kubernetes-sigs/kustomize/pull/3160 - there is no sign of a travis task.

### CODE REVIEW

**Comment by Shell32-Natsu** [comment]
@mikebz Please sign the CLA

**Comment by mikebz** [comment]
@Shell32-Natsu thank you - I think it's done.  I got cncf-cla:yes tag, but the cla/linuxfoundation step on CI/CD is still failing... is there a way to re-trigger that?

File: site/themes/docsy/assets/vendor/bootstrap/README.md [hunk 1]
```diff
@@ -54,7 +54,6 @@ Read the [Getting started page](https://getbootstrap.com/docs/4.1/getting-starte
 ## Status
 
 [![Slack](https://bootstrap-slack.herokuapp.com/badge.svg)](https://bootstrap-slack.herokuapp.com/)
-[![Build Status](https://img.shields.io/travis/twbs/bootstrap/v4-dev.svg)](https://travis-ci.org/twbs/bootstrap)
```
Comment (Shell32-Natsu) [inline site/themes/docsy/assets/vendor/bootstrap/README.md:57]: I think this line is irrelevant from kustomize. It's from dependency.

**Review by Shell32-Natsu — CHANGES_REQUESTED** [review]

File: site/themes/docsy/assets/vendor/bootstrap/README.md (hunk 1 above)
Comment (mikebz) [reply site/themes/docsy/assets/vendor/bootstrap/README.md:57]: good catch.  Let me revert this change

**Pull request merged**